tests/vanilla/store.test.tsx

```diff
@@ -32,3 +32,140 @@ it('should not fire subscription if derived atom value is the same', async () =>
   store.set(countAtom, 1)
   expect(callback).toBeCalledTimes(calledTimes)
 })
+
+describe('dev-only methods', () => {
```

Comment:
```suggestion
describe('[DEV-ONLY] dev-only methods', () => {
```

And, modify `.github/workflows/test-multiple-builds.yml` accordingly.